### DISCUSSION TITLE
give push access to new account

### DISCUSSION THREAD

**Discussion by clhexftw** [ROM-builders/temporary#12087] [General]
With my main gh acc i m unable to build roms in ur ci due to it i created a new account my this github acc is banned in cirrus so

this happens when i build with my main acc: https://cirrus-ci.com/build/4945108549238784

my new acc: @themaltuant
my new acc build: https://cirrus-ci.com/build/4687948238028800

**Comment by rahulkhatri137** [comment]
I thought only my account suspended by Cirrus🙂

**Comment by Apon77** [comment]
> With my main gh acc i m unable to build roms in ur ci due to it i created a new account my this github acc is banned in cirrus so
> 
> this happens when i build with my main acc: https://cirrus-ci.com/build/4945108549238784
> 
> my new acc: @themaltuant my new acc build: https://cirrus-ci.com/build/4687948238028800

there must be some reason for blocking you

**Reply by clhexftw** [reply]
> > With my main gh acc i m unable to build roms in ur ci due to it i created a new account my this github acc is banned in cirrus so
> > this happens when i build with my main acc: https://cirrus-ci.com/build/4945108549238784
> > my new acc: @themaltuant my new acc build: https://cirrus-ci.com/build/4687948238028800
> 
> there must be some reason for blocking you

except using thier ci for building kernels and roms i did nothing else in cirrus
